airstack-modules/package.json

```diff
@@ -4,13 +4,11 @@
   "description": "Integrate Airstack schemas in any subgraph.",
   "main": "modules/airstack/index",
   "scripts": {
-    "codegen": "graph codegen",
-    "build:subgraph": "graph build",
     "build:package": "tsc",
     "lint": "eslint . --ext .ts",
     "test": "echo \"Error: no test specified\" && exit 1",
     "publish": "npm publish --tag alpha --access public",
-    "build": "npm run codegen && npm run build:subgraph && npm run build:package && npm pack"
+    "build": "npm run build:package && npm pack"
```

Comment:
Locally, we will get errors for the missing generated files, 
Please revert this code. 
That was the reason we have graphql and yaml files in the project.

Reply:
Not sure why this is still needed, I have built the module successfully multiple times without this.
Also, without graphql and yaml files, vs code shows errors for imports from ../generated/schema, but when imported inside the subgraph code via npx, it works great.

Reply:
I can understand it doesn't look good in vscode problems section, but I think that is an insufficient reason to keep redundant files inside the codebase. I think it would just confuse devs for the existence of those files.